vite.config.ts

```diff
@@ -17,7 +17,7 @@ export default defineConfig({
         path.resolve(__dirname, "src/utils/reset.css.ts"),
         path.resolve(__dirname, "src/index.ts"),
       ],
-      formats: ["es"],
+      formats: ["es","cjs"],
```

Comment:
I would use this guide here to property enable a cjs build for the lib https://vitejs.dev/guide/build.html#library-mode

Reply:
@marcel-casavo do you mean the parts about the `package.json` entry points? because we are already using the library mode

Reply:
exactly, aligning what is missing with the intended purpose(to have a cjs build)